_opensearch/remote.md

```diff
@@ -132,4 +214,21 @@ curl -X POST "https://localhost:9200/_remotestore/_restore" -ku admin:admin -H '
 ```
 
 If the security plugin is enabled, a user must have the `cluster:admin/remotestore/restore` permission. See [Access control](/security-plugin/access-control/index/) for information about configuring user permissions.
-{: .note}
\ No newline at end of file
+{: .note}
+
+## Potential use cases
+
+You can use remote-backed storage for the following purposes:
+
+- To restore red clusters or indexes
+- To recover all data up to the last acknowledged write regardless of replica count if `index.translog.durability` is set to `request`
+
+## Known limitations
+
+The following are known limitations of the remote-backed storage feature:
+
+- Writing data to a remote store can be a high latency operation when compared to writing data on local file system. This may impact the indexing throughput and latency.
+- Primary to primary relocation is unstable as handover of upload of translogs from older primary to new is yet to be implemented.
```

Comment:
Suggested rewrite: Primary to primary relocation is unstable**,** as **handover of translogs uploads** from older primary to new **hasn't been implemented**.